.golangci.yaml

```diff
@@ -195,6 +195,11 @@ linters-settings:
       - name: var-naming
       - name: waitgroup-by-value
 
+  # irrelevant as of Go v1.22: https://go.dev/blog/loopvar-preview
+  govet:
```

Comment:
nit: I admit I prefer submitting such changes in separate PRs. It is easier to revert if somethings gets broken.

Reply:
The assumption I had in mind was merging the linter change first, and rebase this PR, but it is a nit-pick 👍